### ISSUE TITLE
Fetch with path always appends to file, doesn't overwrite

### ISSUE THREAD

**Issue by membla** [wkh237/react-native-fetch-blob#173]
RNFB: 0.9.6
RN: 0.36.1

```
    RNFetchBlob
    .config({
      path : dirs.DocumentDir + '/assets.xml'
    })
    .fetch('GET', 'http://path/to/assets.xml', {
      //some headers ..
    })
    .progress((received, total) => {
      this.setState({appContainerSourceProgress: received/total})
    })
    .then((res) => {
      console.log('The file saved to ', res.path())
    })
```

If assets.xml exists, new data is appended to the file, not overwritten. Documentation and #141  suggests otherwise.

**Comment by wkh237** [comment]
@membla , thanks for the information. I'll fix the problem 👍

**Comment by wkh237** [comment]
@membla , a fix has launched in `0.10.0-beta.7` please upgrade to this version and try  if it works.

**Comment by membla** [comment]
@wkh237 Thank you, that seems to have solved it.

**Comment by wcandillon** [comment]
@wkh237 I'm using v0.10.2-beta.6 and I have the same issue. Is it possible that this fix is not in v0.10.2-beta.6?
